frontend/src/pages/store/[store].svelte

```diff
@@ -514,8 +515,28 @@
 								maximumFractionDigits: 2,
 							})}
 						</p>
+						{#if item.quantity != null}
+							<p class="text-2xl font-bold">
+								{#if item.quantity == 0}
+									<!-- This is not possible now but it is here if we need it -->
```

Comment:
Why isn't it possible?

Reply:
It wasn't possible to set an item's quantity to zero when I tried

source:
![Screenshot from 2022-01-12 16-49-25](https://user-images.githubusercontent.com/37093293/149235746-d5c5ad17-6ea4-4dc7-a405-8c18aa6c103d.png)

Because of that it isn't possible for an item to ever be out of stock. 
We should probably change that, right?

Reply:
Ah. Will take care of this now.

Reply:
Main branch now allows quantities of 0.